src/pages/settings/AboutPage.js

```diff
@@ -0,0 +1,169 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {View, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'lodash';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import CONST from '../../CONST';
+import {
+    Link, Eye, MoneyBag, Bug, NewWindow,
+} from '../../components/Icon/Expensicons';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import MenuItem from '../../components/MenuItem';
+import Logo from '../../../assets/images/expensify-cash.svg';
+import {version} from '../../../package.json';
+import openURLInNewTab from '../../libs/openURLInNewTab';
+import ONYXKEYS from '../../ONYXKEYS';
+
+
+const propTypes = {
+    // All reports shared with the user
```

Comment:
Please follow our prop documentation style: `/** Whether the IOU is for a single request or a group bill split */`